### ISSUE TITLE
Build custom documentation theme

### ISSUE THREAD

**Issue by jacobtomlinson** [opsdroid/opsdroid#1092]
Currently [our documentation](https://docs.opsdroid.dev/en/stable/) uses the default read the docs theme. It would be great to build our own to make it look more like the [opsdroid website](https://opsdroid.dev/). We could also clean up some of the issues I feel there are with the official theme (code highlighting for one).

We could start with the official theme and extend it [like the Dask project has](https://github.com/dask/dask-sphinx-theme).

This may be related to moving to Sphinx #1090.

**Comment by jacobtomlinson** [comment]
@FabioRosado as you've done awesome work on the website before and are good with web stuff is this something you would be interested in picking up?

**Comment by FabioRosado** [comment]
Yeah I've been meant to pick this up for a while so I would like to take a stab at it for hacktoberfest 😄

**Comment by Cadair** [comment]
SunPy also built a custom sphinx theme, so I would be happy to share what limited experience I have with that.

**Comment by FabioRosado** [comment]
I haven't checked how sphinx works with themes so i will probably poke you before you go on holidays Cadair :D
